addons/source-python/packages/source-python/listeners/tick.py

```diff
@@ -109,19 +109,17 @@ def add(self, delay):
 class Delay(WeakAutoUnload):
     """Execute a callback after a given delay."""
 
-    def __init__(
-        self, delay, callback, cancel_on_map_end=False, *args, **kwargs
-    ):
+    def __init__(self, delay, callback, cancel_on_map_end=False, args=(), kwargs={}):
```

Comment:
This is only true if I store the used value though, in my case I never store it so it's fine and I find it more clear that it's supposed to be a dict.